react/src/layouts/emptyLayout/EmptyLayout.tsx

```diff
@@ -0,0 +1,12 @@
+import { type FC } from 'react';
+import { Outlet } from 'react-router-dom';
+
+export const EmptyLayout: FC = () => {
```

Comment:
what kind of export should we use? default or named? it differs from place to place

Reply:
Only named exports, but for react.lazy need keep default exports for pages-components. I add lazy and suspend in the future